tests/ert_tests/ert3/data/test_transmitters.py

```diff
@@ -139,9 +189,29 @@ async def test_simple_record_transmit_and_load(
     ) as record_transmitter_factory:
         transmitter = record_transmitter_factory(name="some_name")
         await transmitter.transmit_record(record_in)
-
         record = await transmitter.load()
-        assert record.data == expected_data
+        if isinstance(record_in, NumericalRecordTree):
+            sub_records = json.loads(record.data.decode("utf-8"))
+            transmitters = await transmitter.get_recordtree_transmitters(
```

Comment:
Should this looping over the tree be part of the API?

Reply:
Good question.  I would like to have it hidden, but since recordtree requires more data (like subtrasmitters) when loading we need to expose an extra function for creating such subtrasnmitters.